lib/test/unit/ui/console/testrunner.rb

```diff
@@ -396,9 +396,14 @@ def output_statistics(elapsed_time)
               output("Finished in #{elapsed_time} seconds.")
             end
             if @options[:report_slow_tests]
+              output_summary_marker
+              output("Top #{N_REPORT_SLOW_TESTS} slow tests")
               @test_statistics.sort_by {|statistic| -statistic[:elapsed_time]}
                               .first(N_REPORT_SLOW_TESTS)
                               .each do |slow_statistic|
+                left_side = "#{slow_statistic[:name]}: "
+                right_width = @progress_row_max - left_side.size
+                output("#{left_side}%#{right_width}<elapsed_time>f" % slow_statistic)
```

Comment:
In general, we should not mix `#{...}` and `%...` to avoid unexpected formatting. For example, this will be failed when `left_side` has `%s`.

```suggestion
                output("%s%*f" % [
                         left_side,
                         right_width,
                         slow_statistic[:elapsed_time],
                       ])
```

Reply:
Thanks for your suggestion. Certainly, fail with following error.

`%': named<elapsed_time> after unnumbered(1) (ArgumentError)

I fixed in 3a285b0.